scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+# A postprocessing workflow for GATK CNV.
+#
+# THIS CANNOT BE RUN IN TUMOR-ONLY MODE.  A MATCHED NORMAL IS REQUIRED.
+#
+# Internally, we have shown that this workflow improves sensitivity and precision.  Additionally, this workflow is quite
+#  cheap to run.  Note that auxiliary files are required and without these exact files, performance may suffer.
+#
+# UNSUPPORTED and we may modify this WDL before we fully support it.
+#
+# This will also generate absolute (https://software.broadinstitute.org/cancer/cga/absolute) compatible files.
+#  Currently, the balanced-segment calling is extremely ham-fisted.
+#
+# This workflow:
+#  - Does conversion of output seg files to a format compatible with IGV.  This will be removed soon, since the GATK tools
+#     have been updated to produce files that can be easily read by IGV.
+#  - Tags possible germline events that appear in the tumor results.  This is done by comparing breakpoints and
+#     reciprocal overlaps.  This algorithm takes into account any hypersegmentation.
+#  - Tags tumor events that appear in the given blacklists (centromere and gistic blacklist)
+#  - Removes any events that have been tagged in the previous two steps.  Any resulting gaps are covered, so long as
```

Comment:
Perhaps "Any resulting gaps are covered" -> "Any resulting gaps are imputed" would be more clear?

Reply:
Actually, looking a bit further below, I don't think this is good.  If we correctly identify germline events and remove them, I think it is extremely misleading to impute across them.  Why do we need to impute/merge?

Reply:
Practical reasons....  We were worried about downstream tools (GISTIC, ABSOLUTE) having problems.  We have not yet confirmed that these gaps would cause problems, though.

I can file an issue to make this a flag (or the user can specify a max merge distance of zero).  I admit that I am not a fan.